docs/source-pytorch/past_versions.rst

```diff
@@ -0,0 +1,127 @@
+Past PyTorch lightning versions
+===============================
+
+The Pytorch Lightning evolved over time so to keep track, let us share brief version history list with links to particular docs of the given version.
+
+
+.. list-table:: Past versions
+   :widths: 10 50 40
+   :header-rows: 1
+
+   * - Version
+     - Title
+     - Particular Bug-fix versions
+
+   * - `1.9 <https://github.com/Lightning-AI/lightning/releases/tag/1.9.0>`_
```

Comment:
we need to list them from last (e.g. 1.9.3) at the top, to first (1.9.0) at the bottom, within each minor

Reply:
These are in single line